src/mlpack/core/tree/hrectbound_impl.hpp

```diff
@@ -159,6 +159,17 @@ inline void HRectBound<DistanceType, ElemType>::Center(
     center(i) = bounds[i].Mid();
 }
 
+/**
+ * Recompute the minimum width of the bound.
+ */
+template<typename DistanceType, typename ElemType>
+inline void HRectBound<DistanceType, ElemType>::RecomputeMinWidth()
+{
+  minWidth = std::numeric_limits<ElemType>::max();
+  for (size_t i = 0; i < dim; ++i)
+    minWidth = std::min(minWidth, bounds[i].Width());
+}
+
```

Comment:
Yeah, while testing examples I noticed that if you change the bounds manually, the `MinWidth()` bound is wrong (because that is a cached value). So I just added a function to recompute the bound and mentioned it in the documentation.

Reply:
Thanks